README.md

```diff
@@ -12,6 +12,19 @@ You also need the [Grunt](http://gruntjs.com/) build tool:
 
     $ sudo npm install -g grunt-cli
 
+Alternatively, if you dislike installing globally:
+
+    $ npm install grunt-cli
+    $ export PATH=$PWD/node_modules/.bin:$PATH
+    $ type -p grunt-cli ## $PWD/node_modules/.bin/node-cli
+    $ grunt-cli --version
+    grunt-cli v1.2.0
+    grunt v0.4.5
+
```

Comment:
Documentation is fine, though it might be nice to format a block like this with the grey box around it.